developer/src/kmc/src/commands/build.ts

```diff
@@ -64,33 +47,7 @@ File lists can be referenced with @filelist.txt.
 
 If no input file is supplied, kmc will build the current folder.`)
 
-    .action(async (filenames: string[], _options: any, commander: any) => {
```

Comment:
This seems to have largely moved within the new `buildFile` `async` function below... though I'm not sure how it gets triggered.

Reply:
Yes. The `Commander` triggers are not very obvious, but basically the action is fired when the corresponding command is specified on the command line.